### DISCUSSION TITLE
Update Video Src with js code

### DISCUSSION THREAD

**Discussion by jloguercio** [GrapesJS/grapesjs#4113] [Q&A] [answered]
There is a way to update the video src using js code? 

for example i use 

```
component.setAttributes({src: file});
component.updateSrc();
```

when i click preview it doesnt show correctly but if i save and reload the editor it shows good.

Thanks

**Comment by artf** [comment, marked as answer]
Try with `component.set({ src: '...' })`

**Reply by jloguercio** [reply]
It Works !!! Thank you
